Hw_9/hw

```diff
@@ -0,0 +1,187 @@
+    23:25:29.873910 IP 127.0.0.1.42420 > 127.0.0.1.12345: Flags [.], ack 1, win 342,            options [nop,nop,TS val 2958826 ecr 2958826], length 0
+
+  Вот и они. Пакеты, пойманные во время установки соединения
+  
+ 3. Close session. Find in tcpdump output packets that are related to TCP session closure. 
+  
+   Терминал №2: разрываем соединение с TCP-сервером:
+   
+    Ctrl+]
+   Затем выходим из telnet'а:
+    q
+   И смотрим в аутпут tcpdump'а в терминале №3:
+   
+    23:30:28.020371 IP 127.0.0.1.42420 > 127.0.0.1.12345: Flags [F.], seq 1, ack 1, win 342, options [nop,nop,TS val 3256972 ecr 2958826], length 0
+    23:30:28.020561 IP 127.0.0.1.12345 > 127.0.0.1.42420: Flags [F.], seq 1, ack 2, win 342, options [nop,nop,TS val 3256972 ecr 3256972], length 0
+    23:30:28.020571 IP 127.0.0.1.42420 > 127.0.0.1.12345: Flags [.], ack 2, win 342, options [nop,nop,TS val 3256972 ecr 3256972], length 0
+
+  Это пакеты, пойманные во время разрыва соединения
+  
+ 4. run tcpdump and request any http site in separate session. Find HTTP request and answer packets with ASCII data in it.  Tcpdump command must be as strict as possible to capture only needed packages for this http request.
+  
+  Терминал №1:
+   
+   tcpdump -i enp0s3 -nn port 80
+   
+  Терминал №2:
+  
+   curl google.com
+   
+  Смотрим в аутпут терминала №1:
+  
+    23:47:52.365231 IP 10.0.2.15.50506 > 74.125.205.101.80: Flags [S], seq 3713333161, win 29200, options [mss 1460,sackOK,TS val 4301317 ecr 0,nop,wscale 7], length 0
+    23:47:52.376615 IP 74.125.205.101.80 > 10.0.2.15.50506: Flags [S.], seq 14464001, ack 3713333162, win 65535, options [mss 1460], length 0
+    23:47:52.376659 IP 10.0.2.15.50506 > 74.125.205.101.80: Flags [.], ack 1, win 29200, length 0
+    23:47:52.376817 IP 10.0.2.15.50506 > 74.125.205.101.80: Flags [P.], seq 1:75, ack 1, win 29200, length 74: HTTP: GET / HTTP/1.1
+    23:47:52.377157 IP 74.125.205.101.80 > 10.0.2.15.50506: Flags [.], ack 75, win 65535, length 0
+    23:47:52.671435 IP 74.125.205.101.80 > 10.0.2.15.50506: Flags [P.], seq 1:529, ack 75, win 65535, length 528: HTTP: HTTP/1.1 301 Moved Permanently
+    23:47:52.671486 IP 10.0.2.15.50506 > 74.125.205.101.80: Flags [.], ack 529, win 30016, length 0
+    23:47:52.671665 IP 10.0.2.15.50506 > 74.125.205.101.80: Flags [F.], seq 75, ack 529, win 30016, length 0
+    23:47:52.671982 IP 74.125.205.101.80 > 10.0.2.15.50506: Flags [.], ack 76, win 65535, length 0
+    23:47:52.681908 IP 74.125.205.101.80 > 10.0.2.15.50506: Flags [F.], seq 529, ack 76, win 65535, length 0
+    23:47:52.681946 IP 10.0.2.15.50506 > 74.125.205.101.80: Flags [.], ack 530, win 30016, length 0
+
+  "Find HTTP request and answer packets with ASCII data in it"
+  
+   Терминал №1:
+   tcpdump -A -i enp0s3 -nn port 80
```

Comment:
`Tcpdump command must be as strict as possible to capture only needed packages for this http request.`
Your command is not `strict` enough